orte/mca/rml/ofi/rml_ofi_component.c

```diff
@@ -0,0 +1,1091 @@
+/*
+ * Copyright (c) 2015 Intel, Inc. All rights reserved.
```

Comment:
If this is a new file, shouldn't this be copyright 2016?  (I didn't check your other copyrights -- might be worth double checking them)